### PR TITLE
Fixes in control plane services CLI flags

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
- Removed flags that are deprecated and have no effect
   - Unlike daprd where we need to keep flags or older injectors can cause daprd to crash, this isn't a problem for control plane services since they're deployed by Helm
- Remove `tokenAudience` flag from Helm chart since it's now unused
- Fixed description of some flags
- Use constants for default trust anchors file path

### CODE REVIEW

File: cmd/placement/options/options.go [hunk 1]
```diff
@@ -62,8 +63,6 @@ type Options struct {
 	Metrics *metrics.Options
 }
 
-var log = logger.NewLogger("dapr.placement.options")
```
Comment (daixiang0) [inline cmd/placement/options/options.go:65]: No log prefix any more?

Comment (ItalyPaleAle) [reply cmd/placement/options/options.go:65]: There’s nothing writing logs now

**Comment by daixiang0** [comment]
Close this in favor of #7029

**Comment by ItalyPaleAle** [comment]
Per @mukundansundar this should be merged first

**Review by daixiang0 — DISMISSED** [review]
LGTM

File: cmd/sentry/main.go [hunk 2]
```diff
@@ -48,10 +48,6 @@ func main() {
 
 	metricsExporter := metrics.NewExporterWithOptions(log, metrics.DefaultMetricNamespace, opts.Metrics)
 
-	if len(opts.TokenAudience) > 0 {
-		log.Warn("--token-audience is deprecated and will be removed in Dapr v1.14")
```
Comment (mukundansundar) [inline cmd/sentry/main.go:52]: Why is `token-audience` being removed now if it is planned to be removed in 1.14?

File: cmd/operator/options/options.go [hunk 3]
```diff
@@ -72,12 +73,7 @@ func New() *Options {
 	flag.BoolVar(&opts.EnableArgoRolloutServiceReconciler, "enable-argo-rollout-service-reconciler", false, "Enable the service reconciler for Dapr-enabled Argo Rollouts")
 	flag.BoolVar(&opts.WatchdogCanPatchPodLabels, "watchdog-can-patch-pod-labels", false, "Allow watchdog to patch pod labels to set pods with sidecar present")
 
-	flag.StringVar(&opts.TrustAnchorsFile, "trust-anchors-file", "/var/run/secrets/dapr.io/tls/ca.crt", "Path to trust anchors file")
-
-	depCCP := flag.String("certchain", "", "DEPRECATED")
-	depRCF := flag.String("issuer-ca-filename", "", "DEPRECATED")
-	depICF := flag.String("issuer-certificate-filename", "", "DEPRECATED")
-	depIKF := flag.String("issuer-key-filename", "", "DEPRECATED")
+	flag.StringVar(&opts.TrustAnchorsFile, "trust-anchors-file", securityConsts.ControlPlaneDefaultTrustAnchorsPath, "Filepath to the trust anchors for the Dapr control plane")
```
Comment (mukundansundar) [inline cmd/operator/options/options.go:77]: similarly shouldn't this be removed only in 1.14 following the dapr deprecation policy?

**Review by mukundansundar — CHANGES_REQUESTED** [review]

File: cmd/operator/options/options.go (hunk 3 above)
Comment (ItalyPaleAle) [reply cmd/operator/options/options.go:77]: These flags are unused today. This is a K8s service that is only deployed via Helm, so there's no risk of removing flags.

File: cmd/sentry/main.go (hunk 2 above)
Comment (ItalyPaleAle) [reply cmd/sentry/main.go:52]: It's completely unused in this service

Comment (ItalyPaleAle) [reply cmd/sentry/main.go:52]: What I mean is, setting this option today just prints a warning and is a no-op. Should never have said that will be removed, and should have just been removed. This is a control plane service so removing CLI flags is not an issue

**Comment by ItalyPaleAle** [comment]
@mukundansundar @daixiang0 is this ready to be merged?

**Review by JoshVanL — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**